File: cesta/AMarotoChivite-cesta/AMarotoChivite-cesta-Java/src/main/java/simulacionCesta/factories/ProductFactory.java
package simulacionCesta.factories;

import simulacionCesta.models.Product;

import static simulacionCesta.utils.randomData.*;

public class ProductFactory {
    public static Product create() {
        String name = randomProductName();
        double price = randomPrice();
        int quantity = randomQuantity();

        return new Product(name, price, quantity);
    }
}

File: cesta/AMarotoChivite-cesta/AMarotoChivite-cesta-Kotlin/src/main/kotlin/simulacionCesta/factories/ProductFactory.kt
package simulacionCesta.factories

import simulacionCesta.models.Product
import simulacionCesta.utils.randomData

object ProductFactory {
    @JvmStatic
    fun create(): Product {
        val name = randomData.randomProductName()
        val price = randomData.randomPrice()
        val quantity = randomData.randomQuantity()
        return Product(name, price, quantity)
    }
}